spec/reek/smells/irresponsible_module_spec.rb

```diff
@@ -63,14 +48,26 @@ class Bar; end
     expect(src).to reek_of(:IrresponsibleModule)
   end
 
-  it 'reports a fq module name correctly' do
+  it 'reports a class with a trailing comment' do
+    src = <<-EOS
+      class BadClass
+      end # end BadClass
+    EOS
+    expect(src).to reek_of(:IrresponsibleModule)
+  end
+
+  it 'reports a fully qualified class name correctly' do
     src = 'class Foo::Bar; end'
-    ctx = Reek::Context::CodeContext.new(nil, Reek::Source::SourceCode.from(src).syntax_tree)
-    smells = @detector.examine_context(ctx)
-    expect(smells.length).to eq(1)
-    expect(smells[0].smell_category).to eq(described_class.smell_category)
-    expect(smells[0].smell_type).to eq(described_class.smell_type)
-    expect(smells[0].parameters[:name]).to eq('Foo::Bar')
-    expect(smells[0].context).to match(/#{smells[0].parameters['name']}/)
+    expect(src).to reek_of :IrresponsibleModule, name: 'Foo::Bar'
+  end
+
+  context 'when a smell is reported' do
+    before do
+      @bad_module_name = 'WrongUn'
```

Comment:
I believe it was meant to mean 'wrong one' originally. Looks like that whole variable can go, actually.go, actually.